APIs_Platform/CircularBuffer_ex_1/README.md

```diff
@@ -0,0 +1,3 @@
+# CircularBuffer example
+
+This example shows how to use the CircularBuffer APIs. Contents of data stream are pushed until the buffer is full and popped back until the buffer is empty.
```

Comment:
What does it mean for contents to be popped back? Is there another word we should use?

Reply:
It means to remove last element. 
Like here https://en.cppreference.com/w/cpp/container/list/pop_back